gapic/batch/staging.yaml

```diff
@@ -0,0 +1,6 @@
+common:
+  batch_apis: '*'
+  batch_languages: '*'
+  api_config_pattern: ${THISDIR}/../api/artman_*.yaml
+  artman_language_yaml: ${THISDIR}/../lang/common.yaml
+  batch_output_dir: ${REPOROOT}/client-library-staging/generated
```

Comment:
I didn't want to include staging in the name, because of the possibility of setting it to a different and possible non-staging location